Mathlib/Algebra/BigOperators/Group/Finset/Basic.lean

```diff
@@ -831,19 +834,10 @@ theorem prod_dvd_prod_of_dvd {S : Finset α} (g1 g2 : α → β) (h : ∀ a ∈
     rw [Finset.forall_mem_cons] at h
     exact mul_dvd_mul h.1 <| IH h.2
 
-@[to_additive]
-lemma prod_mul_eq_prod_mul_of_exists {s : Finset α} {f : α → β} {b₁ b₂ : β}
-    (a : α) (ha : a ∈ s) (h : f a * b₁ = f a * b₂) :
-    (∏ a ∈ s, f a) * b₁ = (∏ a ∈ s, f a) * b₂ := by
-  classical
-  rw [← insert_erase ha]
-  simp only [mem_erase, ne_eq, not_true_eq_false, false_and, not_false_eq_true, prod_insert]
-  rw [mul_assoc, mul_comm, mul_assoc, mul_comm b₁, h, ← mul_assoc, mul_comm _ (f a)]
-
```

Comment:
same here

Reply:
This one is because I moved `prod_dvd_prod_of_dvd` down as it is not additivisable (we don't have "additive divisibility")